src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        return new Ball((new Random().nextInt(100) + 1), new ColorSupplier().getRandomColor());

```

Comment:
let's create a constant for 100
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names